src/wp-includes/class-wpdb.php

```diff
@@ -902,9 +902,14 @@ public function determine_charset( $charset, $collate ) {
 			}
 		}
 
-		// _unicode_520_ is a better collation, we should use that when it's available.
-		if ( $this->has_cap( 'utf8mb4_520' ) && 'utf8mb4_unicode_ci' === $collate ) {
-			$collate = 'utf8mb4_unicode_520_ci';
+		// _uca1400_ai_ci_ is a better collation, we should use that when it's available.
+		// but fall back to _unicode_520_ when it's available.
```

Comment:
```suggestion
		/*
		 * _uca1400_ai_ci_ is a better collation, we should use that when it's available.
		 * but fall back to _unicode_520_ when it's available.
		 */
```
Use multiline comment formate here.